### PR TITLE
Work in hidden GOPATH dir

### PR DESCRIPTION
**Pull request by zhulik**

### CODE REVIEW

**Comment by mdwhatcott** [comment]
Thanks for working on this. Here are a few issues I can see:

1. Please run `goimports` on the files you've modified to order the import statements.
2. Please also [include test cases](https://github.com/smartystreets/goconvey/blob/master/web/server/watch/functional_core_test.go) that cover the conditions you've added.

**Comment by zhulik** [comment]
`goimports` fixed. About tests: now i have no idea how to test it, i thing, changing GOPATH in runtime is not a good idea, anyway only [one line](https://github.com/zhulik/goconvey/blob/master/web/server/watch/functional_core.go#L39) is not covered in my code

File: web/server/watch/functional_core_test.go [hunk 1]
```diff
@@ -122,6 +123,8 @@ func TestCategorize(t *testing.T) {
 			close(items)
 		}()
 
+		time.Sleep(100 * time.Millisecond)
```
Comment (mdwhatcott) [inline web/server/watch/functional_core_test.go]: Just curious, why is this `time.Sleep` necessary?

**Comment by zhulik** [comment]
Debug time.Sleep removed

**Comment by mdwhatcott** [comment]
The reason that all but one of your new lines are covered is because they happen to be executed by the existing tests, but the result of the behavior you've added is not asserted in any of those test cases. What I'm asking for is that you go back and add assertions (possibly in new test cases) that require your new code to be in place in order to pass.

When adding new functionality we try to first add an additional test that fails because it depends on the new, unwritten behavior. Then, we write the new behavior that makes that test pass. This approach ensures that the production code and the test code stay in sync. It also helps me as the project maintainer feel confident that your additions won't have unexpected (negative) effects on current users. Finally, it prevents someone in the future from modifying the code you've added in a destructive way, breaking it's functionality (even if they did it unknowingly). If you want the code you've added to stand the test of time and remain correct, it needs to be nailed down with assertions.

Here's where the existing test cases live:

https://github.com/smartystreets/goconvey/blob/master/web/server/watch/functional_core_test.go

**Comment by VorT3x** [comment]
@zhulik any update?

**Comment by stefanjarina** [comment]
Hello, any update on this? It is more than year and goconvey web ui still don't run with GVM. Thank you.

**Comment by zhulik** [comment]
@vort3x Sorry, I have already forgotten about this PR. Do I need to make any changes?

**Pull request closed without merging**